docs/src/tutorials/npm-browser-packages/template-deep-dive/src-lib-rs.md

```diff
@@ -63,43 +82,19 @@ Either way, the contents of `utils.rs` define a single public function `set_pani
 
 
 ```rust
-use cfg_if::cfg_if;
+// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
```

Comment:
```suggestion
    // When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
```

Since the code below is also indented by 4 spaces.